src/Console/Commands/BassetNamedAssetsList.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Backpack\Basset\Console\Commands;
+
+use Backpack\Basset\Facades\Basset;
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+
+/**
+ * Basset Named Assets list command.
+ *
+ * @property object $output
+ */
+class BassetNamedAssetsList extends Command
```

Comment:
Is this _really_ only listing named assets? Or is it listing all assets... and if the name isn't set, the URL is the same as the name?